include/onnxruntime/core/graph/graph.h

```diff
@@ -43,11 +43,7 @@
 #include "core/graph/node_arg.h"
 #include "core/graph/ort_format_load_options.h"
 
-namespace flatbuffers {
-class FlatBufferBuilder;
-template <typename T>
-struct Offset;
-}  // namespace flatbuffers
+#include "flatbuffers/flatbuffers.h"
```

Comment:
From here: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

"Include headers in the following order: Related header, C system headers, C++ standard library headers, other libraries' headers, your project's headers."

Please put `#include "flatbuffers/flatbuffers.h"` before the ORT header includes.

Reply:
Fixed